ironfish-cli/src/commands/service/bridge/release.ts

```diff
@@ -0,0 +1,221 @@
+      }
+
+      account = response.content.account.name
+    }
+
+    this.log(`Using account ${account}`)
+
+    while (client.isConnected) {
+      speed.start()
+      speed.reset()
+
+      await this.processNextTransaction(client, account, speed, api)
+    }
+  }
+
+  async processNextTransaction(
+    client: RpcSocketClient,
+    account: string,
+    speed: Meter,
+    api: WebApi,
+  ): Promise<void> {
+    const status = await client.node.getStatus()
+
+    if (!status.content.blockchain.synced) {
+      this.log('Blockchain not synced, waiting 5s')
+      await PromiseUtils.sleep(5000)
+      return
+    }
+
+    if (!status.content.peerNetwork.isReady) {
+      this.log('Peer network not ready, waiting 5s')
+      await PromiseUtils.sleep(5000)
+      return
+    }
+
+    const unprocessedReleaseRequests = await api.getBridgeNextWIronRequests(
+      MAX_RECIPIENTS_PER_TRANSACTION,
+    )
+
+    if (unprocessedReleaseRequests.length === 0) {
+      this.log('No bridge requests, waiting 5s')
+      await PromiseUtils.sleep(5000)
+      return
+    }
+
+    const requestsToProcess = []
+    let totalAmount = 0n
+
+    const response = await client.wallet.getAccountBalance({ account })
+    const availableBalance = BigInt(response.content.available)
+
+    for (const request of unprocessedReleaseRequests) {
+      if (!isValidPublicAddress(request.destination_address)) {
+        // TODO (hughy): submit failed status back to bridge API
+        continue
+      }
+
+      totalAmount += BigInt(request.amount) + 1n
+      if (totalAmount > availableBalance) {
+        this.log(
+          `Bridge account only has available balance for ${requestsToProcess.length} transactions`,
+        )
+        break
+      }
+
+      requestsToProcess.push(request)
+    }
+
+    if (requestsToProcess.length === 0) {
+      this.log('No bridge requests, waiting 5s')
+      await PromiseUtils.sleep(5000)
+      return
+    }
+
+    this.log(
+      `Sending: ${JSON.stringify(
+        requestsToProcess,
+        ['id', 'destination_address', 'amount'],
+        '   ',
+      )}`,
+    )
+
+    const outputs = requestsToProcess.map((req) => {
+      return {
+        publicAddress: req.destination_address,
+        amount: req.amount,
+        memo: req.id,
+        assetId: Asset.nativeId().toString('hex'),
+      }
+    })
+
+    const tx = await client.wallet.sendTransaction({
+      account,
+      outputs,
+      fee: BigInt(requestsToProcess.length).toString(),
+    })
+
+    speed.add(1)
+
+    this.log(
+      `Sent: ${JSON.stringify(
+        requestsToProcess,
+        ['id', 'destination_address', 'amount'],
+        '   ',
+      )} ${tx.content.hash} (5m avg ${speed.rate5m.toFixed(2)})`,
+    )
+
+    const updatePayload = []
+    for (const request of requestsToProcess) {
+      updatePayload.push({
+        id: request.id,
+        destination_transaction: tx.content.hash,
+        status: 'PENDING_ON_DESTINATION_CHAIN',
```

Comment:
What do you think about `PENDING_IRON_RELEASE_TRANSACTION_CONFIRMATION`? I think `PENDING_ON_DESTINATION_CHAIN` makes sense but is less explicit. I might rename or split that out

Reply:
That sounds good to me!